docs/io/optional/callback_example.ipynb

```diff
@@ -1,13 +1,15 @@
 {
  "cells": [
   {
+   "attachments": {},
```

Comment:
Do you know what this change is?

Reply:
@andrewfullard I'm not sure, Jupyter seems to have done it automatically on file save. (I just changed the occurrences of "runner" to "transport", didn't even run any cell.)